django/contrib/admin/options.py

```diff
@@ -843,6 +843,14 @@ def get_changelist(self, request, **kwargs):
 
         return ChangeList
 
+    def get_pagination(self, request, **kwargs):
```

Comment:
Given this returns a class, is there a reason not to use `get_pagination_class` instead of `get_pagination`?
See also comment below in the same area